br/pkg/utils/backoff.go

```diff
@@ -280,3 +284,35 @@ func (bo *pdReqBackoffer) NextBackoff(err error) time.Duration {
 func (bo *pdReqBackoffer) Attempt() int {
 	return bo.attempt
 }
+
+type RawClientBackoffStrategy struct {
```

Comment:
i am a little surprised why we have to make a dedicated rawclientbackoffstrategy in 8.1.

Reply:
every retry has a to have a dedicated strategy if they have different delay parameters, it's the same behavior in master.